service/conf/application.conf

```diff
@@ -93,3 +93,47 @@ application.global=util.Global
 play.http.filters=util.Filters
 play.http.parser.maxMemoryBuffer = 900000K
 play.http.parser.maxDiskBuffer = 900000K
+
+
+opensaber-client {
+  # Default value
+  mapping.file = context_mapping.json
+  registry {
+    service {
+      host = "localhost"
+      port = 8080
+      baseUrl = "http://localhost:8080/"
```

Comment:
How can these values be set as environment variables without having to rebuild for dev / staging / prod?

Reply:
Is this comment closed?

Reply:
Changes has been made to service/conf/application.conf to support environment variables